cameronRomo/silver-nest
# Feature/initial component structure

### What’s this PR do?
The SilverNest App now has component folders, files, and some basic starter UI, including:
- React Browser Router for /about and /get-involved
- Material UI Components for NavBar, Thumbnails (scrolling GridList), and About (image GridList)

The following libraries/dependencies have been installed:
- node-sass (version 4.14.1)
- react-router-dom
- material-ui/core
- CssBaseline from material-ui/core
### Where should the reviewer start?
- App, About, NavBar, Thumbnails line 1
- package.json
- index.js for BrowserRouter and CssBaseline

### How should this be manually tested?
- Run npm start, switch between About and Get Involved tabs to see changes reflected in URL
- View scrollable grid list

### Any background context you want to provide?
N/A

### Screenshots (if appropriate):
![About route/page](https://user-images.githubusercontent.com/68264128/103949169-1509b000-50f8-11eb-85ca-4cd92b63e6bf.png)
![Get Involved route/page](https://user-images.githubusercontent.com/68264128/103949274-3f5b6d80-50f8-11eb-96bc-dbd5eb3c21b0.png)

### Questions:
- How should we render resident thumbnails in Get Involved? A modal, conditional rendering once semester is selected and submitted?
- When/where should we make our API call to retrieve residents data? I'm thinking onSubmit in the Form (once we build out this component)
